2022-03-22 14:40:46,246 INFO    MainThread:91914 [wandb_setup.py:_flush():75] Loading settings from /Users/AFischer/.config/wandb/settings
2022-03-22 14:40:46,246 INFO    MainThread:91914 [wandb_setup.py:_flush():75] Loading settings from /Users/AFischer/PycharmProjects/cocoon-project/docs/wandb/settings
2022-03-22 14:40:46,246 INFO    MainThread:91914 [wandb_setup.py:_flush():75] Loading settings from environment variables: {}
2022-03-22 14:40:46,246 INFO    MainThread:91914 [wandb_setup.py:_flush():75] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2022-03-22 14:40:46,246 INFO    MainThread:91914 [wandb_init.py:_log_setup():405] Logging user logs to /Users/AFischer/PycharmProjects/cocoon-project/docs/wandb/run-20220322_144046-2kistqj5/logs/debug.log
2022-03-22 14:40:46,247 INFO    MainThread:91914 [wandb_init.py:_log_setup():406] Logging internal logs to /Users/AFischer/PycharmProjects/cocoon-project/docs/wandb/run-20220322_144046-2kistqj5/logs/debug-internal.log
2022-03-22 14:40:46,247 INFO    MainThread:91914 [wandb_init.py:init():439] calling init triggers
2022-03-22 14:40:46,247 INFO    MainThread:91914 [wandb_init.py:init():442] wandb.init called with sweep_config: {}
config: {}
2022-03-22 14:40:46,247 INFO    MainThread:91914 [wandb_init.py:init():458] re-initializing run, found existing run on stack: 3jdvq3tz
2022-03-22 14:40:46,248 INFO    MainThread:91914 [wandb_run.py:finish():1539] finishing run afischer/tcn_samp_en-project/3jdvq3tz
2022-03-22 14:40:46,248 INFO    MainThread:91914 [jupyter.py:save_history():429] not saving jupyter history
2022-03-22 14:40:46,248 INFO    MainThread:91914 [jupyter.py:save_ipynb():374] not saving jupyter notebook
2022-03-22 14:40:46,249 INFO    MainThread:91914 [wandb_init.py:_jupyter_teardown():337] cleaning up jupyter logic
2022-03-22 14:40:46,249 INFO    MainThread:91914 [wandb_run.py:_atexit_cleanup():1702] got exitcode: 0
2022-03-22 14:40:46,253 INFO    MainThread:91914 [wandb_run.py:_restore():1674] restore
2022-03-22 14:40:47,070 INFO    MainThread:91914 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 825
  total_bytes: 825
}

2022-03-22 14:40:47,311 INFO    MainThread:91914 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 825
  total_bytes: 825
}

2022-03-22 14:40:47,605 INFO    MainThread:91914 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 825
  total_bytes: 10512
}

2022-03-22 14:40:47,716 INFO    MainThread:91914 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 825
  total_bytes: 10512
}

2022-03-22 14:40:47,832 INFO    MainThread:91914 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 825
  total_bytes: 10512
}

2022-03-22 14:40:47,942 INFO    MainThread:91914 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 10485
  total_bytes: 10512
}

2022-03-22 14:40:48,059 INFO    MainThread:91914 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 10485
  total_bytes: 10512
}

2022-03-22 14:40:48,174 INFO    MainThread:91914 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 10512
  total_bytes: 10512
}

2022-03-22 14:40:48,284 INFO    MainThread:91914 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 10512
  total_bytes: 10512
}

2022-03-22 14:40:48,389 INFO    MainThread:91914 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 10512
  total_bytes: 10512
}

2022-03-22 14:40:48,495 INFO    MainThread:91914 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 10512
  total_bytes: 10512
}

2022-03-22 14:40:48,604 INFO    MainThread:91914 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 10512
  total_bytes: 10512
}

2022-03-22 14:40:49,599 INFO    MainThread:91914 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 10512
  total_bytes: 10512
}

2022-03-22 14:40:50,076 INFO    MainThread:91914 [wandb_run.py:_on_finish():1831] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 10512
  total_bytes: 10512
}
local_info {
}

2022-03-22 14:40:51,232 INFO    MainThread:91914 [wandb_run.py:_footer_history_summary_info():2865] rendering history
2022-03-22 14:40:51,232 INFO    MainThread:91914 [wandb_run.py:_footer_history_summary_info():2894] rendering summary
2022-03-22 14:40:51,234 INFO    MainThread:91914 [wandb_run.py:_footer_sync_info():2822] logging synced files
2022-03-22 14:40:51,237 INFO    MainThread:91914 [wandb_init.py:init():492] starting backend
2022-03-22 14:40:51,237 INFO    MainThread:91914 [backend.py:_multiprocessing_setup():99] multiprocessing start_methods=spawn,fork,forkserver, using: spawn
2022-03-22 14:40:51,239 INFO    MainThread:91914 [backend.py:ensure_launched():219] starting backend process...
2022-03-22 14:40:51,260 INFO    MainThread:91914 [backend.py:ensure_launched():224] started backend process with pid: 92901
2022-03-22 14:40:51,262 INFO    MainThread:91914 [wandb_init.py:init():501] backend started and connected
2022-03-22 14:40:51,285 INFO    MainThread:91914 [wandb_run.py:_label_probe_notebook():947] probe notebook
2022-03-22 14:40:51,285 INFO    MainThread:91914 [wandb_run.py:_label_probe_notebook():957] Unable to probe notebook: 'NoneType' object has no attribute 'get'
2022-03-22 14:40:51,285 INFO    MainThread:91914 [wandb_init.py:init():565] updated telemetry
2022-03-22 14:40:51,383 INFO    MainThread:91914 [wandb_init.py:init():596] communicating run to backend with 30 second timeout
2022-03-22 14:40:52,387 INFO    MainThread:91914 [wandb_run.py:_on_init():1759] communicating current version
2022-03-22 14:40:52,427 INFO    MainThread:91914 [wandb_run.py:_on_init():1763] got version response 
2022-03-22 14:40:52,427 INFO    MainThread:91914 [wandb_init.py:init():625] starting run threads in backend
2022-03-22 14:40:54,691 INFO    MainThread:91914 [wandb_run.py:_console_start():1733] atexit reg
2022-03-22 14:40:54,692 INFO    MainThread:91914 [wandb_run.py:_redirect():1606] redirect: SettingsConsole.WRAP
2022-03-22 14:40:54,692 INFO    MainThread:91914 [wandb_run.py:_redirect():1643] Wrapping output streams.
2022-03-22 14:40:54,693 INFO    MainThread:91914 [wandb_run.py:_redirect():1667] Redirects installed.
2022-03-22 14:40:54,694 INFO    MainThread:91914 [wandb_init.py:init():664] run started, returning control to user process
2022-03-22 14:40:54,695 INFO    MainThread:91914 [wandb_watch.py:watch():47] Watching
2022-03-22 14:42:34,101 INFO    MainThread:91914 [wandb_config.py:__setitem__():153] config set batch_size = 20 - <bound method Run._config_callback of <wandb.sdk.wandb_run.Run object at 0x7fc51be2faf0>>
2022-03-22 14:42:34,101 INFO    MainThread:91914 [wandb_run.py:_config_callback():992] config_cb batch_size 20 None
2022-03-22 14:42:34,102 INFO    MainThread:91914 [wandb_config.py:__setitem__():153] config set epochs = 35 - <bound method Run._config_callback of <wandb.sdk.wandb_run.Run object at 0x7fc51be2faf0>>
2022-03-22 14:42:34,102 INFO    MainThread:91914 [wandb_run.py:_config_callback():992] config_cb epochs 35 None
2022-03-22 14:42:34,102 INFO    MainThread:91914 [wandb_config.py:__setitem__():153] config set lr = 0.005262638047738606 - <bound method Run._config_callback of <wandb.sdk.wandb_run.Run object at 0x7fc51be2faf0>>
2022-03-22 14:42:34,102 INFO    MainThread:91914 [wandb_run.py:_config_callback():992] config_cb lr 0.005262638047738606 None
2022-03-22 14:42:34,102 INFO    MainThread:91914 [wandb_config.py:__setitem__():153] config set no_cuda = False - <bound method Run._config_callback of <wandb.sdk.wandb_run.Run object at 0x7fc51be2faf0>>
2022-03-22 14:42:34,102 INFO    MainThread:91914 [wandb_run.py:_config_callback():992] config_cb no_cuda False None
2022-03-22 14:42:34,102 INFO    MainThread:91914 [wandb_config.py:__setitem__():153] config set seed = 42 - <bound method Run._config_callback of <wandb.sdk.wandb_run.Run object at 0x7fc51be2faf0>>
2022-03-22 14:42:34,102 INFO    MainThread:91914 [wandb_run.py:_config_callback():992] config_cb seed 42 None
2022-03-22 14:44:09,183 INFO    MainThread:91914 [wandb_setup.py:_flush():75] Loading settings from /Users/AFischer/.config/wandb/settings
2022-03-22 14:44:09,183 INFO    MainThread:91914 [wandb_setup.py:_flush():75] Loading settings from /Users/AFischer/PycharmProjects/cocoon-project/docs/wandb/settings
2022-03-22 14:44:09,183 INFO    MainThread:91914 [wandb_setup.py:_flush():75] Loading settings from environment variables: {}
2022-03-22 14:44:09,183 INFO    MainThread:91914 [wandb_setup.py:_flush():75] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2022-03-22 14:44:09,183 INFO    MainThread:91914 [wandb_init.py:_log_setup():405] Logging user logs to /Users/AFischer/PycharmProjects/cocoon-project/docs/wandb/run-20220322_144409-3pjq0w5s/logs/debug.log
2022-03-22 14:44:09,183 INFO    MainThread:91914 [wandb_init.py:_log_setup():406] Logging internal logs to /Users/AFischer/PycharmProjects/cocoon-project/docs/wandb/run-20220322_144409-3pjq0w5s/logs/debug-internal.log
2022-03-22 14:44:09,184 INFO    MainThread:91914 [wandb_init.py:_jupyter_setup():355] configuring jupyter hooks <wandb.sdk.wandb_init._WandbInit object at 0x7fc54e356b50>
2022-03-22 14:44:09,184 INFO    MainThread:91914 [wandb_init.py:init():439] calling init triggers
2022-03-22 14:44:09,184 INFO    MainThread:91914 [wandb_init.py:init():442] wandb.init called with sweep_config: {}
config: {}
2022-03-22 14:44:09,184 INFO    MainThread:91914 [wandb_init.py:init():458] re-initializing run, found existing run on stack: 2kistqj5
2022-03-22 14:44:09,193 INFO    MainThread:91914 [wandb_run.py:finish():1539] finishing run afischer/tcn_samp_en-project/2kistqj5
2022-03-22 14:44:09,194 INFO    MainThread:91914 [wandb_run.py:_atexit_cleanup():1702] got exitcode: 0
2022-03-22 14:44:09,245 INFO    MainThread:91914 [wandb_run.py:_restore():1674] restore
2022-03-22 14:44:09,545 INFO    MainThread:91914 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 825
  total_bytes: 825
}

2022-03-22 14:44:09,764 INFO    MainThread:91914 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 825
  total_bytes: 825
}

2022-03-22 14:44:10,294 INFO    MainThread:91914 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 4
}
pusher_stats {
  uploaded_bytes: 825
  total_bytes: 9473
}

2022-03-22 14:44:10,402 INFO    MainThread:91914 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 825
  total_bytes: 99422
}

2022-03-22 14:44:10,509 INFO    MainThread:91914 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 825
  total_bytes: 99422
}

2022-03-22 14:44:10,617 INFO    MainThread:91914 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 99422
  total_bytes: 99422
}

2022-03-22 14:44:10,728 INFO    MainThread:91914 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 99422
  total_bytes: 99422
}

2022-03-22 14:44:10,832 INFO    MainThread:91914 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 99422
  total_bytes: 99422
}

2022-03-22 14:44:10,938 INFO    MainThread:91914 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 99422
  total_bytes: 99422
}

2022-03-22 14:44:11,045 INFO    MainThread:91914 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 99422
  total_bytes: 99422
}

2022-03-22 14:44:11,153 INFO    MainThread:91914 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 99422
  total_bytes: 99422
}

2022-03-22 14:44:12,746 INFO    MainThread:91914 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 99422
  total_bytes: 99422
}

2022-03-22 14:44:13,382 INFO    MainThread:91914 [wandb_run.py:_on_finish():1831] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 99422
  total_bytes: 99422
}
local_info {
}

2022-03-22 14:44:14,575 INFO    MainThread:91914 [wandb_run.py:_footer_history_summary_info():2865] rendering history
2022-03-22 14:44:14,578 INFO    MainThread:91914 [wandb_run.py:_footer_history_summary_info():2894] rendering summary
2022-03-22 14:44:14,582 INFO    MainThread:91914 [wandb_run.py:_footer_sync_info():2822] logging synced files
